src/ruptures/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""Offline change point detection for Python."""
+
+from .datasets import pw_constant, pw_linear, pw_normal, pw_wavy
+from .detection import Binseg, BottomUp, Dynp, KernelCPD, Pelt, Window
+from .exceptions import NotEnoughPoints
+from .show import display
+
+__version__ = "1.1.2dev0"
```

Comment:
Sure about the `dev0` suffix ?

Reply:
this was a temporary version. Let's change it back to 1.1.2 before merging to master.

Reply:
I will proceed with the change